.enb/lib/page.js

```diff
@@ -0,0 +1,126 @@
+var techs = require('../techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl],
+        [techs.bem.depsOld],
+        [techs.bem.files]
+    ]);
+
+    // css
+    node.addTechs([
+        [techs.css.stylusWithAutoprefixer, { browsers : browsers }],
+        [techs.css.stylus, { target : '?.ie.css', sourceSuffixes : ['styl', 'ie.styl'] }],
+    ]);
+    node.mode('development', function() {
+        node.addTechs([
+            [techs.borschik, { source : '?.css', target : '_?.css', freeze : true, minify : false }],
+            [techs.borschik, { source : '?.ie.css', target : '_?.ie.css', freeze : true, minify : false }]
+        ]);
+    });
+    node.mode('production', function() {
+        node.addTechs([
+            [techs.borschik, { source : '?.css', target : '_?.css', freeze : true, tech : 'cleancss', minify : true }],
+            [techs.borschik, { source : '?.ie.css', target : '_?.ie.css', freeze : true, tech : 'cleancss', minify : true }],
+        ]);
+    });
+    node.addTargets(['_?.css', '_?.ie.css']);
+
+    // JavaScript for browsers
```

Comment:
let's create special helper for depsByTech which encapsulates `bemdecl` -> `deps` -> `files` steps:

```js
function getFilesTechByDepsByTech(sourceTech, targetTech) {
    return [
        [techs.bem.depsByTechToBemdecl, {
            target : '?.' + targetTech + '.bemdecl.js',
            sourceTech : sourceTech,
            destTech : targetTech
        }],
        [techs.bem.depsOld, {
            target : '?.' + targetTech + '.deps.js',
            bemdeclFile : '?.' + targetTech + '.bemdecl.js'
        }],
        [techs.bem.files, {
            depsFile : '?.' + targetTech + '.deps.js',
            filesTarget : '?.' + targetTech + '.files',
            dirsTarget : '?.' + targetTech + '.dirs'
        }]       
    ];
}
```